src/engraving/tests/split_data/split01.mscx

```diff
@@ -1,38 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<museScore version="4.00">
+<museScore version="4.10">
   <Score>
     <Division>480</Division>
     <Style>
-      <Spatium>1.76389</Spatium>
+      <Spatium>1.74978</Spatium>
       </Style>
     <showInvisible>1</showInvisible>
     <showUnprintable>1</showUnprintable>
     <showFrames>1</showFrames>
-    <showMargins>0</showMargins>
+    <showMargins>0</showMargins>    
     <metaTag name="arranger"></metaTag>
     <metaTag name="composer"></metaTag>
-    <metaTag name="copyright"></metaTag>
-    <metaTag name="lyricist"></metaTag>
+    <metaTag name="copyright"></metaTag>   
+     <metaTag name="lyricist"></metaTag>
     <metaTag name="movementNumber"></metaTag>
-    <metaTag name="movementTitle"></metaTag>
+    <metaTag name="movementTitle"></metaTag>    
     <metaTag name="poet"></metaTag>
     <metaTag name="source"></metaTag>
+    <metaTag name="sourceRevisionId"></metaTag>
     <metaTag name="translator"></metaTag>
     <metaTag name="workNumber"></metaTag>
     <metaTag name="workTitle">Test</metaTag>
-    <Part>
+    <Part id="1">
       <Staff id="1">
         <StaffType group="pitched">
           <name>stdNormal</name>
           </StaffType>
         </Staff>
       <trackName>Voice</trackName>
-      <Instrument>
```

Comment:
What is the reason for this change? (and the similar ones)

Reply:
These tests should be updated since it currently incorrectly adds a C Major key signature to the second measure after splitting the measure when it shouldn't modify the key signature. 

Frankly, I'm not sure how to make the reference files properly (without the unnecessary metadata etc), so I just remade both the before and after (ref) files again, which resulted in the different instruments, part ids, etc. Let me know if I should do something else instead.

Reply:
The core of the problem is actually that these test files follow an old version of the file format, because they were never updated; and, apart from that just nobody has made the effort, the reason that they were never updated, is that saving mscx files (rather than mscz, which are binary files and thus not usable for comparisons) has become a lot more cumbersome in MS4. 

So, usually we just look if the differences between the ref xml and the result of the test make sense, and then edit the ref file accordingly... 

Anyway, the changes mostly seem fine, I've only a few requests:
- It would make more sense to change the instrument IDs to voice, actually like it was in the ref files
- There are some undesired whitespace changes in the non-ref files, like spaces added/removed at the beginning/end of the line. Would be great if you could fix those.

Reply:
Makes sense. Didn't realize all the awful whitespaces I added to the files, that's my bad.
I've reverted the tests back to what's in master currently, and only removed the key sigs from the ref xml.